### ISSUE TITLE
Allow setting endpoint for SES

### ISSUE THREAD

**Issue by jeznag** [terraform-providers/terraform-provider-aws#7985]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

There is currently no way to set the endpoint for SES in the aws provider block: https://www.terraform.io/docs/providers/aws/

This makes it impossible to use localstack with terraform templates that contain SES resources.

### New or Affected Resource(s)

* aws_XXXXX

### Potential Terraform Configuration

```hcl
provider aws {
  region  = "ap-southeast-2"
  version = "~> 1.27"

  skip_credentials_validation = true
  skip_metadata_api_check     = true
  s3_force_path_style         = true
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
  endpoints {
    ses     = "http://localhost:4579"
  }
}
```

### References

#3888

**Comment by bflad** [comment]
Support for setting the SES endpoint of the provider has been merged and will release with version 2.3.0 of the Terraform AWS Provider, likely middle of this week. 👍 We'll probably get the few other LocalStack supported endpoints customizable as well in the future, but a feature request for those as a reminder would be fantastic.

**Comment by bflad** [comment]
This has been released in [version 2.3.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#230-march-21-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
